### PR TITLE
Release v3.3.2

### PR DESCRIPTION
**Pull request by jhelison**
# Contents of the Release

- fix: add crossbeam-epoch dependency to fix panic backtrace messages on 0.7.1 version #52
- V3 custom config for testnets #57
- allow usernet to mine more blocks using header version 6, just like f… #58
- fix: A small update to enable block mining on usernet v3 #59
- feat: usernet now is optimized to mine more blocks per second #60
- fix: small fix related to match_ref variable #61
- Fix/documentation #62
- Feat/get blocks #64
- feat: Added feedback if foundations are broken #65
- Update the develop branch with master commits #66
- Feat: Add Testnet configuration file #67
- Floonet configuration #68
- Fix cucumber tests #69
- Freeze randomx and progpow libraries using tags #73
- Make builds reproducible #74
- Edit pre-commit and add PR template #75
- Add automation to build the release binaries and tar files #76
- Update main readme for the project #78

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,18 +1,30 @@
 [package]
 name = "epic"
-version = "3.0.0"
+version = "3.3.2"
 authors = ["Epic Foundation <mimblewimble@lists.launchpad.net>"]
 description = "Simple, private and scalable cryptocurrency implementation based on the MimbleWimble chain format."
 license = "Apache-2.0"
 repository = "https://github.com/mimblewimble/epic"
```
Comment (Jualns) [inline Cargo.toml:7]: Change this repository to "https://github.com/EpicCash/epic"

File: api/Cargo.toml [hunk 2]
```diff
@@ -1,11 +1,11 @@
 [package]
 name = "epic_api"
-version = "3.0.0"
+version = "3.3.2"
 authors = ["Epic Foundation <mimblewimble@lists.launchpad.net>"]
 description = "APIs for epic, a simple, private and scalable cryptocurrency implementation based on the MimbleWimble chain format."
 license = "Apache-2.0"
 repository = "https://github.com/mimblewimble/epic"
```
Comment (Jualns) [inline api/Cargo.toml:7]: Change this repository to "https://github.com/EpicCash/epic"

File: chain/Cargo.toml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "epic_chain"
-version = "3.0.0"
+version = "3.3.2"
 authors = ["Epic Foundation <mimblewimble@lists.launchpad.net>"]
 description = "Chain implementation for epic, a simple, private and scalable cryptocurrency implementation based on the MimbleWimble chain format."
 license = "Apache-2.0"
```
Comment (Jualns) [inline chain/Cargo.toml:6]: Change this repository to "https://github.com/EpicCash/epic"

File: config/Cargo.toml [hunk 4]
```diff
@@ -1,11 +1,11 @@
 [package]
 name = "epic_config"
-version = "3.0.0"
+version = "3.3.2"
 authors = ["Epic Foundation <mimblewimble@lists.launchpad.net>"]
 description = "Configuration for epic, a simple, private and scalable cryptocurrency implementation based on the MimbleWimble chain format."
 license = "Apache-2.0"
 repository = "https://github.com/mimblewimble/epic"
```
Comment (Jualns) [inline config/Cargo.toml:7]: Change this repository to "https://github.com/EpicCash/epic"

File: keychain/Cargo.toml [hunk 5]
```diff
@@ -1,11 +1,11 @@
 [package]
 name = "epic_keychain"
-version = "3.0.0"
+version = "3.3.2"
 authors = ["Epic Foundation <mimblewimble@lists.launchpad.net>"]
 description = "Chain implementation for epic, a simple, private and scalable cryptocurrency implementation based on the MimbleWimble chain format."
 license = "Apache-2.0"
 repository = "https://github.com/mimblewimble/epic"
```
Comment (Jualns) [inline keychain/Cargo.toml:7]: Change this repository to "https://github.com/EpicCash/epic"

File: p2p/Cargo.toml [hunk 6]
```diff
@@ -1,11 +1,11 @@
 [package]
 name = "epic_p2p"
-version = "3.0.0"
+version = "3.3.2"
 authors = ["Epic Foundation <mimblewimble@lists.launchpad.net>"]
 description = "Chain implementation for epic, a simple, private and scalable cryptocurrency implementation based on the MimbleWimble chain format."
 license = "Apache-2.0"
 repository = "https://github.com/mimblewimble/epic"
```
Comment (Jualns) [inline p2p/Cargo.toml:7]: Change this repository to "https://github.com/EpicCash/epic"

File: servers/Cargo.toml [hunk 7]
```diff
@@ -1,11 +1,11 @@
 [package]
 name = "epic_servers"
-version = "3.0.0"
+version = "3.3.2"
 authors = ["Epic Foundation <mimblewimble@lists.launchpad.net>"]
 description = "Simple, private and scalable cryptocurrency implementation based on the MimbleWimble chain format."
 license = "Apache-2.0"
 repository = "https://github.com/mimblewimble/epic"
```
Comment (Jualns) [inline servers/Cargo.toml:7]: Change this repository to "https://github.com/EpicCash/epic"

File: src/bin/epic.rs [hunk 8]
```diff
@@ -82,6 +83,18 @@ fn real_main() -> i32 {
 		.get_matches();
 	let node_config;
 
+
+    // run floonet and usernet with csutom PolicyConfig
```
Comment (Jualns) [inline src/bin/epic.rs:87]: "custom"

File: store/Cargo.toml [hunk 9]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "epic_store"
-version = "3.0.0"
+version = "3.3.2"
 authors = ["Epic Foundation <mimblewimble@lists.launchpad.net>"]
 description = "Simple, private and scalable cryptocurrency implementation based on the MimbleWimble chain format."
 license = "Apache-2.0"
```
Comment (Jualns) [inline store/Cargo.toml:6]: Change this repository to "https://github.com/EpicCash/epic"

File: util/Cargo.toml [hunk 10]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "epic_util"
-version = "3.0.0"
+version = "3.3.2"
 authors = ["Epic Foundation <mimblewimble@lists.launchpad.net>"]
 description = "Simple, private and scalable cryptocurrency implementation based on the MimbleWimble chain format."
 license = "Apache-2.0"
```
Comment (Jualns) [inline util/Cargo.toml:6]: Change this repository to "https://github.com/EpicCash/epic"

**Review by Jualns — DISMISSED** [review]
LGTM!

File: src/bin/epic.rs (hunk 8 above)
Comment (jhelison) [reply src/bin/epic.rs:87]: Done

File: servers/Cargo.toml (hunk 7 above)
Comment (jhelison) [reply servers/Cargo.toml:7]: Done

File: p2p/Cargo.toml (hunk 6 above)
Comment (jhelison) [reply p2p/Cargo.toml:7]: Done

File: config/Cargo.toml (hunk 4 above)
Comment (jhelison) [reply config/Cargo.toml:7]: Done

File: chain/Cargo.toml (hunk 3 above)
Comment (jhelison) [reply chain/Cargo.toml:6]: Done

File: api/Cargo.toml (hunk 2 above)
Comment (jhelison) [reply api/Cargo.toml:7]: Done

File: Cargo.toml (hunk 1 above)
Comment (jhelison) [reply Cargo.toml:7]: Done

File: core/src/core/block/feijoada.rs [hunk 11]
```diff
@@ -82,6 +82,170 @@ pub struct PolicyConfig {
 	pub policies: Vec<Policy>,
 }
 
+impl PolicyConfig {
```
Comment (paultergust) [inline core/src/core/block/feijoada.rs:85]: I see a lot of repetition on this block. We should refactor this when possible.

**Review by paultergust — APPROVED** [review]

**Review by Jualns — APPROVED** [review]
LGTM!

**Pull request merged**